apis/config/v1alpha1/clientconfig.go

```diff
@@ -19,6 +25,10 @@ const (
 	ExperimentalUnstableVersions VersionSelectorLevel = "experimental"
 	// NoUnstableVersions allows no unstable plugin versions, format major.minor.patch only
 	NoUnstableVersions VersionSelectorLevel = "none"
+	// DefaultTCECompatibilityFileLocation is the OCI location for TCE' compatibility file
```

Comment:
nit: should `DefaultTCECompatibilityFileLocation` be updated to `DefaultTCEBomRepo`